From 4cd7d9b073971412caa1e65789cc7af903a4abd2 Mon Sep 17 00:00:00 2001
From: tedyu <yuzhihong@gmail.com>
Date: Wed, 15 Apr 2015 05:04:32 -0700
Subject: [PATCH 207/224] HBASE-13475 Small spelling mistake in
 region_mover#isSuccessfulScan causes NoMethodError
 (Victor Xu)

Reason: Bug
Author: Victor Xu
Ref: CDH-30248
---
 bin/region_mover.rb |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/region_mover.rb b/bin/region_mover.rb
index 565b0d5..134b032 100644
--- a/bin/region_mover.rb
+++ b/bin/region_mover.rb
@@ -100,7 +100,7 @@ def isSuccessfulScan(admin, r)
   scan = Scan.new(r.getStartKey(), r.getStartKey())
   scan.setBatch(1)
   scan.setCaching(1)
-  scan.setFilter(FilterList.new(FirstKeyOnlyFilter.new(),InclusiveStopFilter().new(r.getStartKey())))
+  scan.setFilter(FilterList.new(FirstKeyOnlyFilter.new(),InclusiveStopFilter.new(r.getStartKey())))
   begin
     table = HTable.new(admin.getConfiguration(), r.getTableName())
     scanner = table.getScanner(scan)
-- 
1.7.9.5

